samples/dll/sdk_exe.cpp

```diff
@@ -65,7 +65,7 @@ OnCommand(HWND /* hwnd */, int id, HWND /* hwndCtl */, UINT /* codeNotify */)
     }
 }
 
-void OnDestroy(HWND hwnd)
+void OnDestroy(HWND /* hwnd */)
```

Comment:
We normally use `WXUNUSED()`, but this is minor, of course.

Reply:
Not available. This file does not use wxWidgets. This style to silence the warning is already used in the file.

Reply:
Ah, yes, of course, sorry for failing to notice this.